src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-param-reassign */
+import React from 'react';
+import { PersonType } from '../types';
+import { Person } from './Person';
+
+type Props = {
+  people: PersonType[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => (
+  <table
+    data-cy="peopleTable"
+    className="table is-striped is-hoverable is-narrow is-fullwidth"
+  >
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Sex</th>
+        <th>Born</th>
+        <th>Died</th>
+        <th>Mother</th>
+        <th>Father</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      {people.map(person => {
+        if (person.motherName) {
+          person.mother = people.find(p => p.name === person.motherName);
+        }
```

Comment:
please don't call a function from the template because it will be executed every time when the state change, so make sense to move it to the variable